### ISSUE TITLE
Nightlies in F-Droid

### ISSUE THREAD

**Issue by plafue** [jffrymrtn/writeily-pro#128]
Based on feedback from #1, i will learn a bit more about the f-droid release process and provide some re-branded nightlies for early adopters.

Some ideas:
* Another colour for the icon (i don't know, red, maybe?) or with some kind of clear distinction from the oficcial writeily ( :exclamation: , :boom: , :bomb: or similar danger icon as badge)
  * The rest of the UI design will remain the same
* A clear disclaimer that this is bleeding edge, e.g.:
  * *Using this writeily version may delete all your files, eat your puppy and burn down your house. You are on your own.*

**Comment by dufferzafar** [comment]
Yeah, please provide nightlies. A lot of the people using this app, I feel are developers themselves, so we'll try to help you in terms of bug reports and all.

Also, I just like living on the bleeding edge, when it comes to software :)

**Comment by plafue** [comment]
Well, from now on I'm trying to push towards continuous delivery, so maybe with some luck I won't need to create nightlies... I'll keep you guys posted :). Any feedback is very much appreciated.

**Comment by dufferzafar** [comment]
Is there somewhere I can download the apk of the app from the master branch?

**Comment by plafue** [comment]
Not yet, although you can build use app yourself quite easily. 
Right now the last apk you can find under releases is pretty close to head

**Comment by dufferzafar** [comment]
Yeah, that'll totally do. For some reason, I didn't even look at the releases. Thanks a bunch. :smile:

**Comment by plafue** [comment]
That release is available in fdroid and at the play store :)
